starknet/src/utils/types.cairo

```diff
@@ -6,14 +6,88 @@ use hash::LegacyHash;
 use option::OptionTrait;
 use clone::Clone;
 use starknet::{
-    ContractAddress, Store, contract_address_const, StorageBaseAddress, SyscallResult,
+    EthAddress, ContractAddress, Store, contract_address_const, StorageBaseAddress, SyscallResult,
     storage_write_syscall, storage_read_syscall, storage_address_from_base_and_offset,
     storage_base_address_from_felt252, contract_address::Felt252TryIntoContractAddress,
     syscalls::deploy_syscall, class_hash::Felt252TryIntoClassHash
 };
 use sx::utils::math::pow;
 use array::SpanTrait;
 
+#[derive(Copy, Drop, Serde, LegacyHash, PartialEq, starknet::Store)]
```

Comment:
I believe `LegacyHash` should not be used when possible. Are we sure we wish to derive it?

Reply:
removed